### ISSUE TITLE
egg hatching and double down on travel distance

### ISSUE THREAD

**Issue by HibikiFox** [j-e-k/poketrainer#480]
```
2016-08-09 17:03:28,012 [fort_walker] [ INFO] Traveled 941.42 meters of 9237.00 of the trip
2016-08-09 17:03:28,012 [fort_walker] [ INFO] Traveled 941.42 meters of 9237.00 of the trip
```

traveled distance is duplicated each line, and egg hatching is no longer reported, if it is even working... no errors yet.. but no way to see distance walked on eggs

**Comment by freakyfreak87** [comment]
Im getting a LOT less visited/spun forts on my end same with egg hatching

**Comment by destiny117** [comment]
I have just merged #485 
- it should solve the duplicate lines for your setup
- it should make egg-hatching work again

> Im getting a LOT less visited/spun forts

I suspect the reason is as follows: we have more console output atm because we are more closely following the step_size limit, so you have just more output during forts spun, check the new config options to change the logging behavior

Also remember that there might have been new speed limits implemented, so make sure your step_size is not too high

will close this now, please open a new issue if something is still not working as you'd expect
